auth/authpb/auth.proto

```diff
@@ -16,7 +16,21 @@ message User {
   int64 tombstone = 3;
 }
 
+// Permission is a single entity
+message Permission {
+  bytes path = 1;
+
+  enum PermissionType {
+    READ = 0;
+    WRITE = 1;
+  }
+  int32 permissionBitmap = 2;
+}
+
 // Role is a single entry in the bucket authRoles
 message Role {
-  bytes name = 2;
+  bytes name = 1;
+
+  repeated Permission pathPermission = 2;
```

Comment:
path (it should be key) is an exact name of a path. prefix is a prefix of (typically) multiple paths. I thought setting permissions of ranged keys can be provided with prefix. Should we provide another kind of permission (e.g. a range permission that has a number of keys).